letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -444,8 +457,18 @@ if [ "$1" = "--le-auto-phase2" ]; then
 
     echo "Installing Python packages..."
     TEMP_DIR=$(TempDir)
+    trap "rm -rf $TEMP_DIR" EXIT
```

Comment:
I'll fix the single quotes, but I don't think EXIT is bash specific ([source](http://www.unix.com/man-page/POSIX/1posix/trap/)).

I tested the script on FreeBSD and the directory was deleted.